core/src/main/java/org/lint/azzert/command/ExemptDisabledTestsCommand.java

```diff
@@ -12,6 +12,11 @@ public Void execute(final Context context){
         context.getMethodContexts().removeIf(
                 method -> ! Collections.disjoint(context.getSupportedExemptApis(), method.getAnnotations())
         );
+
```

Comment:
would've made sense to fold method.isIgnored() into the filter above. also, this seems to be a different approach to identifying disabled tests, will take a detailed look later.

Reply:
is ignored is a flag which is added when enabled is false for TestNG, merging the filters